### PR TITLE
Mention Multimesh custom data will be compressed to half in Compatibility

### PR DESCRIPTION
**Pull request by jsjtxietian**
For https://github.com/godotengine/godot/issues/93875

### CODE REVIEW

**Comment by Mickeon** [comment]
If you "Note" it should be put into a `[b]Note[/b]` like many messages of this nature.
Also may be worth documenting in more places other than this one setter (such as the corresponding getter, but there are likely more)

File: doc/classes/MultiMesh.xml [hunk 1]
```diff
@@ -65,7 +65,7 @@
 			<param index="0" name="instance" type="int" />
 			<param index="1" name="custom_data" type="Color" />
 			<description>
-				Sets custom data for a specific instance. [param custom_data] is a [Color] type only to contain 4 floating-point numbers.
+				Sets custom data for a specific instance. [param custom_data] is a [Color] type only to contain 4 floating-point numbers. Note each number is stored in 32 bits in the Forward+ and Mobile rendering methods, but is packed into 16 bits in Compatibility.
```
Comment (clayjohn) [inline doc/classes/MultiMesh.xml:68]:
```suggestion
				Sets custom data for a specific instance. [param custom_data] is a [Color] type only to contain 4 floating-point numbers. [b]Note:[/b] Each number is stored in 32 bits in the Forward+ and Mobile rendering methods, but is packed into 16 bits in the Compatibility rendering method.
```

Comment (jsjtxietian) [reply doc/classes/MultiMesh.xml:68]: Thanks!

File: doc/classes/MultiMesh.xml [hunk 2]
```diff
@@ -65,7 +65,7 @@
 			<param index="0" name="instance" type="int" />
 			<param index="1" name="custom_data" type="Color" />
 			<description>
-				Sets custom data for a specific instance. [param custom_data] is a [Color] type only to contain 4 floating-point numbers.
+				Sets custom data for a specific instance. [param custom_data] is a [Color] type only to contain 4 floating-point numbers. [b]Note:[/b] Each number is stored in 32 bits in the Forward+ and Mobile rendering methods, but is packed into 16 bits in the Compatibility rendering method.
```
[the remark this answers is not in the document]
Comment (jsjtxietian) [reply doc/classes/MultiMesh.xml:68]: Thanks!

**Review by Mickeon — APPROVED** [review]
This is nice as is, but I still stand by:

> Also may be worth documenting in more places other than this one setter (such as the corresponding getter, but there are likely more)

**Comment by Lielay9** [comment]
I'd copy the note for instance-color too. It could also be mentioned in the buffer, but since it has no description atm, it's not as important.

**Review by clayjohn — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**